grovel/grovel.lisp

```diff
@@ -802,8 +802,12 @@ string."
     (process-wrapper-form out form)))
 
 (define-wrapper-syntax in-package (name)
-  (setq *package* (find-package name))
-  (push `(in-package ,name) *lisp-forms*))
+  (let ((desired-package (find-package name)))
+    (assert desired-package ()
+            "Wrapper file specified (in-package ~s)~%however that does not name a known package"
```

Comment:
more diverse error messages are easier to search for. probably not a big deal as the users are programmers and should get a backtrace, but still...